### ISSUE TITLE
Algorithm to decide whether a repository's status is RED, ORANGE, or GREEN.

### ISSUE THREAD

**Issue by suryagowda** [fluttercommunity/admin_dashboard#8]
1. Defining the criteria for each status. For example, defining RED as a repository that has multiple open issues or pull requests that are marked as critical or urgent, ORANGE as a repository that has a few open issues or pull requests that are marked as important or moderate, and GREEN as a repository that has no open issues or pull requests marked as critical, urgent, important, or moderate.
2. Then , retrieving the data for the repository. This might include the number and severity of open issues and pull requests, the number of subscribers or contributors, the activity level of the repository (e.g. number of commits, issues, and pull requests in the past week), and any other relevant data points.
3. Then, based on the defined criteria and the retrieved data, assigning a status to the repository. If the repository meets the criteria for RED, assign it a RED status. If it meets the criteria for ORANGE, assign it an ORANGE status. If it meets the criteria for GREEN, assign it a GREEN status.
Finally, we have to consider implementing some logic to handle cases where the repository doesn't fit neatly into one of the three defined categories. For example, we might have a threshold for determining whether a repository is ORANGE or GREEN, or we might want to assign a special status if the repository has no open issues or pull requests at all.
Can i go ahead with this and write the code?

**Comment by abdelrahmanmagdii** [comment]
Yes I believe this part of the code can be implemented with the state of the app at the moment.

**Comment by MabudAlam** [comment]
Any update on this project @abdelrahmanmagdii  @mhadaily ?

**Comment by mhadaily** [comment]
This project is not up to date, and we are not actively working on it anymore.

**Comment by MabudAlam** [comment]
> This project is not up to date, and we are not actively working on it anymore.

Any plans to work on this during GSoC 2024?
